python/tk_multi_loader/delegate_publish_history.py

```diff
@@ -95,18 +213,9 @@ def _on_before_paint(self, widget, model_index, style_options):
         except:
             pass
             
-            
         # set the little description bit next to the artist icon
-        if sg_item.get("description") is None:
-            desc_str = "No Description Given"
-        else:
-            desc_str = sg_item.get("description")
-        
-        if sg_item.get("created_by") is None:
-            author_str = "Unspecified User"
-        else:
-            author_str = "%s" % sg_item.get("created_by").get("name")
-
+        desc_str = sg_item.get("description") or "No Description Given"         
+        author_str = sg_item.get("created_by").get("name") or "Unspecified User"
```

Comment:
I think `sg_item.get("created_by", {}).get("name")` would give a false sense of security, because it would be returned as `None` from shotgun if the value isn't defined but the field is requested! The code does make the assumption that `created_by` always exist - I think this is set for every record, whatever you do in Shotgun. I thought it was a safe assumption to make because created_by is such a special field, like creation_time, updated_by etc (I think they are called audit fields!). Does that make sense? Happy to put in additional checks, but only if you absolutely feel we need to!